olive/passes/onnx/conversion.py

```diff
@@ -177,3 +187,18 @@ def _run_for_config(
 
         # save the model to the output path and return the model
         return model_proto_to_olive_model(onnx_model, output_model_path, config)
+
+
+class DeviceSpecificOnnxConversion(OnnxConversion):
```

Comment:
With this pass, we will have different search path for different devices with different caches right?

Reply:
Yes